data_summaries/Capybara.json

```diff
@@ -0,0 +1,570 @@
+{
+    "capybara-Dove": {
+        "Unique Dataset Identifier": "capybara-Dove",
+        "Dataset Name": "capybara_Dove",
+        "Paper Title": "",
+        "Dataset URL": "https://huggingface.co/datasets/LDJnr/Capybara",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/LDJnr/Capybara",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "Capybara",
+        "Collection URL": "https://huggingface.co/datasets/LDJnr/Capybara",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Dialog Generation",
+            "Explanation Generation",
+            "Instruction Following",
+            "Instructional Question Answering"
+        ],
+        "Text Sources": [
+            "sharegpt.com",
+            "chat.lmsys.org"
+        ],
+        "Model Generated": ["OpenAI GPT-4"],
```

Comment:
~where does it say this? I can't find any statement on the HF page about which model(s) generated the data~ ah it's because of the GPT-4 data in Pure-Dove, right?

Reply:
Yeah, it's because of Pure-Dove